FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,12 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] response in
+			guard self != nil else { return }
+			switch response {
+			case .success(let result): completion(FeedImageMapper.map(data: result.0, from: result.1))
```

Comment:
Suggestion:
`result` is a tuple and thus it can be decomposed right in the case description. It will make the code a little bit more descriptive and easier to read. Otherwise one probably will go to `FeedLoader` and try to figure out what the `result`'s type is.

```suggestion
			case let .success((data, response)): completion(FeedImageMapper.map(data: data, from: response))
```